### ISSUE TITLE
Tray icon improvements

### ISSUE THREAD

**Issue by ldepandis** [stanfieldr/ghetto-skype#59]
It think it would be better for the tray icon if its color could match the current status (gray for invisible, perhaps?), just like in Skype for Linux.
Also, it could be even more useful if the tray icon could show the number of unread chats.

**Comment by quangtt** [comment]
Agreed.
But it seems like ghetto-skype doesn't have
But I'm using GNOME instead of Unity, could you guys consider bringing tray icon to GNOME for others like me?

**Comment by Pulfer** [comment]
Indeed, it would be major improvement. Maybe not a number but at least some notification about unread chats on tray icon.

**Comment by Pulfer** [comment]
> Maybe not a number but at least some notification about unread chats on tray icon.

Seems to be already implemented in current git snapshot.
